src/index.php

```diff
@@ -1,5 +1,7 @@
 <?php
 
+// Hello
+
 /**
```

Comment:
```suggestion
/**
```

Should be removed once the testing done. Right?

Reply:
Yep! Thanks.